### ISSUE TITLE
https://vuex.vuejs.org gives 404s for every asset.

### ISSUE THREAD

**Issue by LukeusMaximus** [vuejs/vuejs.org#1715]
Every asset under https://vuex.vuejs.org/assets/ returns a 404 for me. The HTML file loads however. This wasn't the case yesterday.

**Comment by chrisvfritz** [comment]
@LukeusMaximus While `vuex.vuejs.org/assets` 404s, because no page exists there, I'm not seeing the referenced assets themselves 404. Is that the case for you?

**Comment by LukeusMaximus** [comment]
Ok, I've resolved this with a cache clear after discovering it worked fine in an incognito tab. Did you add that service worker recently? I am otherwise baffled.

**Comment by chrisvfritz** [comment]
I don't believe it was recently. Weird! 😄 Thanks for reporting though and we'll definitely reopen if it happens again.

**Comment by LukeusMaximus** [comment]
Ok, so this has actually recurred. A whole bunch. Every time I open one of the links in the sidebar I get the same thing: an unstyled page with no JS. The network tab of chrome devtools tells me that all css and js resources (under that `/assets` path) have responded with 404. However, after doing an "Empty cache and hard reload" the problem disappears. The problem promptly reoccurs when I open a different link, or a reopen a link previously visited.

I'm inclined to believe "it's just me" as neither my phone or any other machine exhibits this problem. In fact, incognito tabs on the same machine never exhibit the problem either. But if I'm the problem, why is the server sending 404s? The connection is fine (I get the HTML and all the content within) and secure (Green padlocks are your friend) so I can hardly attribute the fault to something occurring in the middle.

When I get back to the office, I'll attempt to get the dump of one of these failing requests... unless the problem vanishes entirely in my weekend's absence.

**Comment by LukeusMaximus** [comment]
I've captured both good and bad HAR files. Is there somewhere I can upload these for you (preferably somewhere private)?

There's one thing I've noticed in the bad HAR: the requests have some odd headers starting with ":" such as ":scheme", ":method", and ":authority". Looking at the good HAR, there are far fewer headers involved on the same requests. Also the URL paths are slightly different for some reason.
